Cargo.toml

```diff
@@ -146,45 +147,45 @@ xshell = "0.2"
-cairo-lang-runner = { git = "https://github.com/starkware-libs/cairo", rev = "67c6eff9c276d11bd1cc903d7a3981d8d0eb2fa2" }
-cairo-lang-semantic = { git = "https://github.com/starkware-libs/cairo", rev = "67c6eff9c276d11bd1cc903d7a3981d8d0eb2fa2" }
-cairo-lang-sierra = { git = "https://github.com/starkware-libs/cairo", rev = "67c6eff9c276d11bd1cc903d7a3981d8d0eb2fa2" }
-cairo-lang-sierra-ap-change = { git = "https://github.com/starkware-libs/cairo", rev = "67c6eff9c276d11bd1cc903d7a3981d8d0eb2fa2" }
-cairo-lang-sierra-gas = { git = "https://github.com/starkware-libs/cairo", rev = "67c6eff9c276d11bd1cc903d7a3981d8d0eb2fa2" }
-cairo-lang-sierra-generator = { git = "https://github.com/starkware-libs/cairo", rev = "67c6eff9c276d11bd1cc903d7a3981d8d0eb2fa2" }
-cairo-lang-sierra-to-casm = { git = "https://github.com/starkware-libs/cairo", rev = "67c6eff9c276d11bd1cc903d7a3981d8d0eb2fa2" }
-cairo-lang-sierra-type-size = { git = "https://github.com/starkware-libs/cairo", rev = "67c6eff9c276d11bd1cc903d7a3981d8d0eb2fa2" }
-cairo-lang-starknet = { git = "https://github.com/starkware-libs/cairo", rev = "67c6eff9c276d11bd1cc903d7a3981d8d0eb2fa2" }
-cairo-lang-starknet-classes = { git = "https://github.com/starkware-libs/cairo", rev = "67c6eff9c276d11bd1cc903d7a3981d8d0eb2fa2" }
-cairo-lang-syntax = { git = "https://github.com/starkware-libs/cairo", rev = "67c6eff9c276d11bd1cc903d7a3981d8d0eb2fa2" }
-cairo-lang-syntax-codegen = { git = "https://github.com/starkware-libs/cairo", rev = "67c6eff9c276d11bd1cc903d7a3981d8d0eb2fa2" }
-cairo-lang-test-plugin = { git = "https://github.com/starkware-libs/cairo", rev = "67c6eff9c276d11bd1cc903d7a3981d8d0eb2fa2" }
-cairo-lang-test-runner = { git = "https://github.com/starkware-libs/cairo", rev = "67c6eff9c276d11bd1cc903d7a3981d8d0eb2fa2" }
-cairo-lang-test-utils = { git = "https://github.com/starkware-libs/cairo", rev = "67c6eff9c276d11bd1cc903d7a3981d8d0eb2fa2" }
-cairo-lang-utils = { git = "https://github.com/starkware-libs/cairo", rev = "67c6eff9c276d11bd1cc903d7a3981d8d0eb2fa2" }
+cairo-lang-casm = { git = "https://github.com/starkware-libs/cairo", rev = "03ead1c4a7bdacdbcfd79ab65aa05a7e76c28e01" }
+cairo-lang-compiler = { git = "https://github.com/starkware-libs/cairo", rev = "03ead1c4a7bdacdbcfd79ab65aa05a7e76c28e01" }
+cairo-lang-debug = { git = "https://github.com/starkware-libs/cairo", rev = "03ead1c4a7bdacdbcfd79ab65aa05a7e76c28e01" }
+cairo-lang-defs = { git = "https://github.com/starkware-libs/cairo", rev = "03ead1c4a7bdacdbcfd79ab65aa05a7e76c28e01" }
+cairo-lang-diagnostics = { git = "https://github.com/starkware-libs/cairo", rev = "03ead1c4a7bdacdbcfd79ab65aa05a7e76c28e01" }
+cairo-lang-doc = { git = "https://github.com/starkware-libs/cairo", rev = "03ead1c4a7bdacdbcfd79ab65aa05a7e76c28e01" }
+cairo-lang-executable = { git = "https://github.com/starkware-libs/cairo", rev = "03ead1c4a7bdacdbcfd79ab65aa05a7e76c28e01" }
```

Comment:
We've forgot to patch it before, see https://github.com/software-mansion/scarb/pull/1857